examples/create-react-app-with-typescript/src/node-utils.ts

```diff
@@ -0,0 +1,164 @@
+function toCommonProperties(items: any) {
```

Comment:
I think having this function might make the example harder to understand - why is it converting the properties instead of just defining the properties directly?

Reply:
this is part of one of the RFCs, and also just a convenience for me to make it easier to change and create different properties quickly. One of the open PRs adds a more robust version of this function as part of `@elyra/pipeline-services`